src/db/migrations/28-add-mighty-33-others-die.ts

```diff
@@ -17,7 +17,7 @@ const MIGHTY_33: MusicPlatform = {
 
 const OTHERS_DIE: NftFactory = {
   id: '0x97b9f21b41041e344f5bd71e3e86b69e79dcc0a6',
-  startingBlock: '10876005',
+  startingBlock: '10876004',
```

Comment:
hm looks like this contract was actually created on block 10688108?

See https://etherscan.io/tx/0x69a998a7b1290d015b84682d727b70f686b8277297b902a1f7112c717820d48a

Also, I guess since we won't pickup this past change automatically, in order to actually deploy this we'll need to either:
 - delete/drop this NFTFactory and all it's cascading nfts/tracks as part of a new migration
 - delete/drop this NFTFactory and all it's cascading nfts/tracks just manually when we deploy
 - rebuild whole db from scratch and replace with new rebuild

lmk whichever you think is quickest/easiest

Reply:
Oh damn! I've been getting the starting block based on the earliest transaction that's taken place within that contract, i.e. https://etherscan.io/txs?a=0x97b9f21b41041e344f5bd71e3e86b69e79dcc0a6

Moving forward I'll make sure to look at the contract inception block (and add it to my guide).

Reply:
RE: deployment
I think we should do a full rebuild given that we want to pick up the proper +1 and -1's for each transfer between collectors, for all contracts on the system since the beginning of indexing.

I did a full rebuild on the QA environment earlier, so I think we can just dump that and update the repo?

Reply:
sounds good

Reply:
> Oh damn! I've been getting the starting block based on the earliest transaction that's taken place within that contract, i.e. https://etherscan.io/txs?a=0x97b9f21b41041e344f5bd71e3e86b69e79dcc0a6
> 
> Moving forward I'll make sure to look at the contract inception block (and add it to my guide).

yep exactly - we should use the deployment block

Reply:
Ok, I've double checked the previous contracts I had added and they're all correct. Will push up the fix for this block along with the other changes.